src/aw_watcher_input/main.py

```diff
@@ -3,26 +3,21 @@
 from time import sleep
 
 import aw_client
-import click
 from aw_core import Event
 from aw_watcher_afk.listeners import KeyboardListener, MouseListener
 
 logger = logging.getLogger(__name__)
 
-
-@click.command()
-@click.option("--testing", is_flag=True)
-def main(testing: bool):
+def main(args, testing: bool):
```

Comment:
Please don't remove click. Rewrite to use it instead.

@ellipsis-dev try to do this for us.

Reply:
no worries. wasnt sure what was the new and what was the old